src/app/create/page.tsx

```diff
@@ -1,60 +1,182 @@
-"use client"
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
+"use client";
 
-import { AdapterDayjs } from '@mui/x-date-pickers/AdapterDayjs';
-import { LocalizationProvider } from '@mui/x-date-pickers/LocalizationProvider';
-import { ThemeProvider, createTheme } from '@mui/material/styles';
-import { yupResolver } from '@hookform/resolvers/yup';
-import { useForm } from 'react-hook-form';
-import * as yup from 'yup';
-import { CarpoolDetails } from '@/components/carpoolDetails';
+import { AdapterDayjs } from "@mui/x-date-pickers/AdapterDayjs";
+import { LocalizationProvider } from "@mui/x-date-pickers/LocalizationProvider";
+import { ThemeProvider, createTheme } from "@mui/material/styles";
+import { yupResolver } from "@hookform/resolvers/yup";
+import { useForm } from "react-hook-form";
+import * as yup from "yup";
+import { CarpoolDetails } from "@/components/carpoolDetails";
+import { createEvent } from "../actions";
+import { toast } from "sonner";
+import { useRef, useState } from "react";
+import copy from "copy-text-to-clipboard";
 
 interface IFormInputs {
-    event_name: string,
-    location: string,
-    date: Date,
-    time: Date
+  event_name: string;
+  address: string;
```

Comment:
sorry for not renaming this xd

didnt test it since the forntend was disconnected so u caught it when u tested

Reply:
np